src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PETypeParameterSymbol.cs

```diff
@@ -212,6 +212,10 @@ private ImmutableArray<TypeSymbolWithAnnotations> GetDeclaredConstraintTypes()
                             continue;
                         }
 
+                        if (moduleSymbol.UtilizesNullableReferenceTypes)
+                        {
+                            typeSymbol = NullableTypeDecoder.TransformType(TypeSymbolWithAnnotations.Create(typeSymbol), constraintHandle, moduleSymbol).TypeSymbol;
```

Comment:
>typeSymbol = NullableTypeDecoder.TransformType(TypeSymbolWithAnnotations.Create(typeSymbol), constraintHandle, moduleSymbol).TypeSymbol; [](start = 28, length = 136)

Dropping top level nullability?

Reply:
Yes. Add a `// PROTOTYPE` comment.

---
In reply to: [194567984](https://github.com/dotnet/roslyn/pull/27589#discussion_r194567984) [](ancestors = 194567984)